src/pages/staking/StakeForm.vue

```diff
@@ -155,17 +155,21 @@ export default {
         MetaMaskLogo,
         displayConfirmModal: false,
         resultHash: null,
-        header: 'Stake TLOS',
-        subheader: 'Staking your TLOS to sTLOS grants you access to a steady income and various DeFi applications, ' +
+        harcoded_header: 'Stake TLOS',
```

Comment:
why are some of these left as "hardcoded"?

Reply:
It was a helper I used to "teach" my Github copilot plugin which uses AI to suggest code ahead.

Changing all the `label`(s) for `harcoded_label`(s) and immediately after the first one, writing the correct translation solution, my copilot learned from it and write the rest for me.

But I just forgot to remove them.

Done ([735c2d2](https://github.com/telosnetwork/teloscan/pull/292/commits/735c2d2f6ddcd35dbada25260a3b4c48425873e8))